proofs/eo/cpc/rules/Nfa.eo

```diff
@@ -0,0 +1,14 @@
+(include "../programs/Nfa.eo")
```

Comment:
As mentioned, let's omit `rules/Nfa.eo`.  I will replace the call to your method on a separate PR.